src/rlookup.c

```diff
@@ -379,7 +379,7 @@ void RLookup_Cleanup(RLookup *lk) {
   memset(lk, 0xff, sizeof(*lk));
 }
 
-static RSValue *hvalToValue(const RedisModuleString *src, RLookupCoerceType type) {
+RSValue *hvalToValue(const RedisModuleString *src, RLookupCoerceType type) {
```

Comment:
Does removing the statics impact the performance in any way?

Reply:
Removed the `static` to make it an available symbol "externally"?